imdclient/IMDProtocol.py

```diff
@@ -30,10 +30,11 @@ class IMDHeaderType(Enum):
     # New in IMD v3
     IMD_SESSIONINFO = 10
     IMD_RESUME = 11
-    IMD_TIME = 12
-    IMD_BOX = 13
-    IMD_VELOCITIES = 14
-    IMD_FORCES = 15
+    IMD_WAIT = 12
```

Comment:
This isn't the order of the packet types in the protocol, if you change this it will break all of our implementations